charter/index.html

```diff
@@ -288,7 +288,7 @@ <h3>
             If additional in-scope Recommendation-track deliverables need to be added to the Charter before the Charter expires, the Working Group will prepare an updated Charter that differs only in deliverables.
           </p>
           <p>
-            The Working Group will not adopt new proposals until they have matured through the <a href="https://www.w3.org/groups/cg/solid">W3C Solid Community Group</a> or another similar incubation phase.
+            The Working Group aims to adopt proposals it deems sufficiently mature; such proposals can be submitted through the <a href="https://www.w3.org/groups/cg/solid">W3C Solid Community Group</a> or other incubation phase.
```

Comment:
Pavlik's or Wouter's suggestions [[1](https://github.com/solid/solid-wg-charter/pull/48#issuecomment-1691568434)], [[2](https://github.com/solid/solid-wg-charter/pull/48#issuecomment-1691601128)] both acknowledge prior work, understanding of incubation, and BP at W3C, and would be adequate to approve the PR.

```suggestion
            The Working Group will use the [W3C Recommendation Track Readiness Criteria](https://www.w3.org/Guide/standards-track/#criteria) when evaluating proposals. The Working Group will not adopt new proposals until they have matured through the <a href="https://www.w3.org/groups/cg/solid">W3C Solid Community Group</a> or another similar incubation phase.
```